packages/next/src/build/webpack-config.ts

```diff
@@ -847,7 +847,6 @@ export default async function getBaseWebpackConfig(
         rootDir: dir,
         pagesDir,
         appDir,
-        hasServerComponents,
```

Comment:
hasServerComponents is still used as a option in next-swc-loader

Reply:
Yes, what's wrong with that?

Reply:
ah okay it's only applied for server layer compilcation